researchstudio-sat/webofneeds
# Automatic commit-hook setup for prettier

During `npm install` a commit-hook will be created by husky, that ensures that all staged files are formatted.

Test:

1. make sure you've run `npm install`
2. in a javascript-file (e.g. app_jspm.js) make a change with a really long line of stuff, no trailing comma, etc
3. stage and commit that file => it should be formatted
4. remove that commit (e.g. via `git rebase --interactive`)
